src/Test/Utilities/Portable/Compilation/CompilationExtensions.cs

```diff
@@ -251,5 +251,23 @@ internal static bool CanHaveExecutableCodeBlock(ISymbol symbol)
             }
         }
 
+        public static void CheckOperations(this Compilation compilation)
```

Comment:
>CheckOperations [](start = 27, length = 15)

It looks like this code no longer uses TestOperationWalker, I think we still should try accessing all the properties from each node. I guess we can add this in a separate PR